.github/workflows/golang-security-scan.yml

```diff
@@ -29,20 +35,76 @@ on:
         type: string
         default: "[]" # empty array
         description: Json-formatted array of strings to pass to the docker build command as build args. e.g. '["arg1", "arg2"]'
+      scan-latest-release:
+        description: "Scan the latest release of the repository in addition to the default branch"
+        default: true
+        type: boolean
```

Comment:
It looks like this is being required... do we want to require this? 

I was thinking about a drop-in replacement without causing any BC brakage. If it defaults to true, and we do not need to specify, then upgrades would start scanning the latest release automatically. I feel like that would be a safe default and if we explicity do not want to scan the latest release, then we could disable it.

I am trying to decide what would be best.

Reply:
But, I see that the DEPENDABOT_ALERTS_TOKEN will case a BC brakage anyway, so maybe that part isn't a big deal.

Reply:
Added 'required: false' per our discussion.